app/assets/stylesheets/provider/_authentication_providers.scss

```diff
@@ -1,18 +0,0 @@
-.fa-auth0:before {
```

Comment:
So, I decided to remove this. `@extend .fa-star` was not working (because of font-awesome dependency change), and moreover, for a long time I thought that having this was quite stupid.

Like - why is it even a good idea to show a generic star icon as a "logo" of Auth0??
<img width="424" height="193" alt="Screenshot From 2025-09-16 11-40-34" src="https://github.com/user-attachments/assets/0421374a-f335-4949-baf1-5976559a1c69" />

Or a `❮❯` text as a "logo" of RH-SSO? (like - yeah, because it's similar to Keycloak's logo...).

By the way, GitHub's logo is already **not** shown in production, not sure at what point it got broken, but well, I think GitHub doesn't need a logo either, the name is enough.